domain/modelconfig/modelmigration/import.go

```diff
@@ -73,6 +74,24 @@ func (i *importOperation) Execute(ctx context.Context, model description.Model)
 		return errors.NotValidf("model config")
 	}
 
+	// Models imported from older controllers may contain config attributes
+	// which have since been removed from use. We filter these out by removing
+	// any incoming attributes not in the default list.
+	defaults, err := i.defaultsProvider.ModelDefaults(ctx)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	defaultAttrs := set.NewStrings()
+	for k := range defaults {
+		defaultAttrs.Add(k)
+	}
+
+	for k := range attrs {
+		if !defaultAttrs.Contains(k) {
+			delete(attrs, k)
```

Comment:
We need to log that we have discarded data (what key and value was discarded).

Reply:
done